charts/galoy/values.yaml

```diff
@@ -593,8 +593,14 @@ oathkeeper:
       mutators:
         noop:
           enabled: true
+        id_token:
+          enabled: true
+          config:
+            jwks_url: file:///etc/secrets/mutator.id_token.jwks.json
+            issuer_url: "galoy.io"
       access_rules:
         matching_strategy: regexp
+    mutatorIdTokenJWKs: '{ "keys": [ { "use": "sig", "kty": "RSA", "kid": "ad3cab84-6f62-45c4-abee-6146b1257e8c", "alg": "RS256", "n": "plXws2h_LudAleQ16jTX7lkwPxzOk9ud-Ti_Qi00Qk9AMY5dKWQGRSpI2OHTSahX7QdN_0DfMlIjM55o_l1cIltMEPt8Bllns8v6VKIlaOk4BzkYKf6pc1hTyTEWaVw2gh7vYyurycfQ-7e0BNToZ0QVL6s-rSjVm_cCvkRx1uVVLRFWroBYk-rWN1nC50Pucr6A3sm_jphY7Jzt9KFUijNRtHTBPN11NqUTU7apklI_-sU-x8_jBBrB2Hb3PNkpvjqXT8ug-C7M5yYltBIAL08Gsce-aAX55nCFFRICVrGv-gKMwwdMjQQMEU-yRWvW2doc48fy9ubk4vtHd0xbWQ", "e": "AQAB", "d": "pBmFLnqZfxyGi1pqFnWr0moLPzicVSMF5yolBpSySfkFy9436u3TgckFtXKVkMWUeTQLrxfBG1QNho9x5gEVNI_9q4NwCJOLy8YhYf3WCnqL3BRn0oZ7hmBNyN-0zJ086YK7iYs8IAYSF0RVRa8G2VpBzJWVcCApHYXEHxJHfUmMXpaETBacmgxTheiRpSPCpLHifhWpq6khHWLMHaaXZnIFii6CGhCc_gbYtTZm_jAcC_KL5SP27qabgM50aTzBSBwRs0kvADvvlaOUI-5qelrXig-MbrFGeMsA7H0HJojAlQU2gyiw295MH0n9g1F75HkICSuZQbERh2Keqw9QQQ", "p": "zDkd6W44QcdkcwZR_a_yZj5FDIz1D-YtciKcnPIE3ZHRqPe6w-0Exyd8DRd-OURuGjZWCXSGJ0hDFZo0ClOR13EhOMJRLkSM1Cpzbg3nyrpmbrMZdg6IUx556Wa_Tl86mhuyjfWSdL2yBUjZPLTu2wB_xj5oyROGpTWiEiEZsOM", "q": "0IHIFV-Y06UFqyYmRoYa9NFgRpSiKjx0_K6-tf1HbZsnxcRQBMDabQDkIrH-K83U1ulPYA59YTIATcIOAclYYm-y9LI98GpbJXTYj_Y64FeOX_k0XshdEZ0h5skAdZ2SK9wit6Bq0VR8oT3TyZP5-k4WsZva0ILaH2XvlC6MY5M", "dp": "g-jkrDFYPwymkjFj5kOug2H3QasOY02ezxkSJcGhaOhWDxisVv8SVfzoAKDLE_xiTaYJHqRYPqlLQ3nN4VmnWwwklD_XJdZ7TFN7KXvr2yNZJGmkqX-riFyoDB0ktl-GP6JiS5VyGtuoUAeE4dRQb767yXXoQvDMyIuC-sjMilc", "dq": "aGEP8OMydoEeHVezwYpQ5E4MjEjMBajOR9CS2Dg7EbuiZXapODfzIK5ZhqTkQLmB2zRg8A1Q5t9JF6QVrv9Ep8KTK7wvFbOofU_JOghh4Pzz4Ue319sl_DNVTVtCR57iFeLV3mcIElLhMTbqVvclJRECe1cyaz-WQjGdhejnjtE", "qi": "DYS6rZvq90A9OVsKzknNGxATwQEVK1dKpeuGXgkuseT5N7Tb36htFaHu2k0j1WNHArdZYf215S7d0kYECyyOIGsOL2QNkwrg0cIiQCQdUot-9oNgD3QfxQNJrmcAJkNL5XJEv9_yVVcoKh2yy898LfbQZymcjs3xRlQV1EBEUy8" } ] }'
```

Comment:
I'm not sure that should be here? this is a private key

maybe this is just for test? but the key is also present at `file:///etc/secrets/mutator.id_token.jwks.json` afaiu

Reply:
This is for the default OOB deploy. Obviously in production we don't want this. Can discuss at standup how to make the OOB deploy work.